README.md

```diff
@@ -151,32 +151,43 @@ target:
 It is worth mentioning that you can use Harbor robot accounts using OCI registries as source or target.
 
 Also, take into account that if you use OCI as the source repository you must specify the list of charts to synchronize
-or a pointer to an [index file](#helm-charts-index-for-oci-based-repositories) in the repository.
+or a pointer to a [remote index file](#remote-index-for-oci-based-repositories) in the repository.
 
-#### Helm Charts Index for OCI-based repositories
+#### Remote index for OCI-based repositories
 
-By using a charts index file for OCI-Based repository you won't need to maintain a hardcoded list of chart names in the config file.
+By using a remote index file for OCI-Based repository you won't need to maintain a hardcoded list of chart names in the config file.
 charts-syncer will be able to auto-discover what charts need to be synchronized.
 
 ~~~yaml
 source:
  repo:
    kind: OCI
    url: https://my-oci-registry.io/my-project/subpath
-   chartsIndex: my-oci-registry.io/my-project/index:latest
+   # Enable remote index lookup
+   useRemoteIndex: true
```

Comment:
I still believe that charts index is a better name, it indicates what the index will contain and at the end of the day `remote` is implicit in the URI